commit 14b1f4168122e06138b2abb88c31934159dd0796
Author: Laurent Bigonville <bigon@bigon.be>
Date:   Thu Jun 27 21:37:19 2013 +0200

    Add calls to dh_systemd_enable and dh_systemd_start
    
    dh_systemd_enable and dh_systemd_start must be called in that order
    before and after dh_installinit call

diff --git a/1/rules/debhelper.mk.in b/1/rules/debhelper.mk.in
index 5a0f6fb..bf88ec1 100644
--- a/1/rules/debhelper.mk.in
+++ b/1/rules/debhelper.mk.in
@@ -152,7 +152,9 @@ _cdbs_deprecated_vars += DEB_UPDATE_RCD_PARAMS DEB_STRIP_EXCLUDE DEB_COMPRESS_EX
 # New in 0.4.85.
 DEB_DH_INSTALL_MENU_ARGS_DEFAULT ?= $(DEB_DH_INSTALL_MENU_ARGS)
 DEB_DH_INSTALL_CRON_ARGS_DEFAULT ?= $(DEB_DH_INSTALL_CRON_ARGS)
+DEB_DH_SYSTEMD_ENABLE_ARGS_DEFAULT ?= $(DEB_DH_SYSTEMD_ENABLE_ARGS)
 DEB_DH_INSTALLINIT_ARGS_DEFAULT ?= $(DEB_DH_INSTALLINIT_ARGS)
+DEB_DH_SYSTEMD_START_ARGS_DEFAULT ?= $(DEB_DH_SYSTEMD_START_ARGS)
 DEB_DH_INSTALLDEBCONF_ARGS_DEFAULT ?= $(DEB_DH_INSTALLDEBCONF_ARGS)
 DEB_DH_INSTALLEMACSEN_ARGS_DEFAULT ?= $(DEB_DH_INSTALLEMACSEN_ARGS)
 DEB_DH_INSTALLCATALOGS_ARGS_DEFAULT ?= $(DEB_DH_INSTALLCATALOGS_ARGS)
@@ -208,7 +210,9 @@ $(patsubst %,binary-install/%,$(DEB_ALL_PACKAGES)) :: binary-install/%:
 	dh_installinfo -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_INSTALL_INFO) 
 	dh_installmenu -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_DH_INSTALL_MENU_ARGS)
 	dh_installcron -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_DH_INSTALL_CRON_ARGS)
+	$(if $(wildcard /usr/bin/dh_systemd_enable),dh_systemd_enable -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_DH_SYSTEMD_ENABLE_ARGS))
 	dh_installinit -p$(cdbs_curpkg) $(if $(call cdbs_expand_curvar,DEB_UPDATE_RCD_PARAMS),--update-rcd-params="$(call cdbs_strip_quotes,$(call cdbs_expand_curvar,DEB_UPDATE_RCD_PARAMS))") $(call cdbs_expand_curvar,DEB_DH_INSTALLINIT_ARGS)
+	$(if $(wildcard /usr/bin/dh_systemd_start),dh_systemd_start -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_DH_SYSTEMD_START_ARGS))
 	dh_installdebconf -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_DH_INSTALLDEBCONF_ARGS)
 	dh_installemacsen -p$(cdbs_curpkg) $(if $(DEB_EMACS_PRIORITY),--priority=$(DEB_EMACS_PRIORITY)) $(if $(DEB_EMACS_FLAVOR),--flavor=$(DEB_EMACS_FLAVOR)) $(call cdbs_expand_curvar,DEB_DH_INSTALLEMACSEN_ARGS)
 	dh_installcatalogs -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_DH_INSTALLCATALOGS_ARGS)
